### PR TITLE
Manual lint fix base master

### PR DESCRIPTION
**Pull request by ruuushhh**

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    # E131 continuation line unaligned for hanging indent
+    E131,
+    # E129 visually indented line with same indent as next logical line
+    E129,
+    # Multiple spaces after ',' (E241)
+    E241
+max-line-length = 99
+max-complexity = 19
+ban-relative-imports = true
+select = B,C,E,F,N,W,I25
```
Comment (ashwin1111) [inline .flake8:30]: what does select do?

File: apps/quickbooks_online/models.py [hunk 2]
```diff
@@ -31,7 +31,7 @@ def get_transaction_date(expense_group: ExpenseGroup) -> str:
 
 
 def get_expense_purpose(workspace_id, lineitem, category, workspace_general_settings) -> str:
-    fyle_credentials = FyleCredential.objects.get(workspace_id=workspace_id)
+    FyleCredential.objects.get(workspace_id=workspace_id)
```
Comment (ashwin1111) [inline apps/quickbooks_online/models.py:34]: let's remove this line, not needed

File: tests/test_mappings/test_tasks.py [hunk 3]
```diff
@@ -109,65 +140,6 @@ def test_disable_category_for_items_mapping(db ,mocker):
...
-
-def test_disable_category_for_items_mapping(db, mocker):
-    workspace_id = 3
-    mocker.patch(
-        'fyle_integrations_platform_connector.apis.Categories.sync',
-        return_value=[]
-    )
-    mocker.patch(
-        'fyle_integrations_platform_connector.apis.Categories.post_bulk',
-        return_value=[]
```
Comment (ashwin1111) [inline tests/test_mappings/test_tasks.py:138]: why are these removed?

Comment (ruuushhh) [reply tests/test_mappings/test_tasks.py:138]: this was repeating test case

File: apps/quickbooks_online/models.py (hunk 2 above)
Comment (ruuushhh) [reply apps/quickbooks_online/models.py:34]: okay

File: .flake8 (hunk 1 above)
Comment (ruuushhh) [reply .flake8:30]: it lets you decide which checks to run and which to ignore

**Review by ashwin1111 — APPROVED** [review]

File: apps/fyle/helpers.py [hunk 4]
```diff
@@ -99,13 +85,14 @@ def get_cluster_domain(refresh_token: str) -> str:
...
+    join_by = None
     for expense_filter in expense_filters:
         constructed_expense_filter = construct_expense_filter(expense_filter)
-        
+
         # If this is the first filter, set it as the final filter
         if expense_filter.rank == 1:
-            final_filter = (constructed_expense_filter)
-        
+            final_filter = constructed_expense_filter
```
[the remark this answers is not in the document]
Comment (ashwin1111) [reply apps/fyle/helpers.py:94]: yeah no issues

**Review by ashwin1111 — APPROVED** [review]
